### PR TITLE
[Merged by Bors] - feat: improve smoke tests visibility (IN-1101)

### PR DESCRIPTION
**Pull request by edison-vflow**
### Fixes or implements 
* [IN-1101](https://linear.app/voiceflow/issue/IN-1101/improve-logs-in-our-cicd-system)

### Successful run 
* https://app.circleci.com/pipelines/gh/voiceflow/general-runtime/16323/workflows/7dc9898c-138b-4b7e-a71d-b2e206b4c216
### Brief description. What is this change?
* The change allows us to 
  * Capture component logs during a smoke test run 
     * Logs are `tailed` for the duration of the smoke test run allowing us to preserve logs from initial pod instance if a 
     pod restart occurs
   * Capture summary pod state for all pods before smoke test run 
   * Capture summary pod state for all pods after smoke test run 
   * Capture detailed pod state for each pod after smoke test run 

### Related PR's 
* https://github.com/voiceflow/orb-common/pull/250
* https://github.com/voiceflow/infrastructure/pull/685

### Checklist

- [x] Logs collect on a successful run 
- [x] Logs collect on a failed run

### CODE REVIEW

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

**Comment by edison-vflow** [comment]
bors try

File: .circleci/continue_config.yml.tmpl [hunk 1]
```diff
@@ -43,14 +43,14 @@ defaults:
...
-    tag: {{ .values.node_version }}-vf-1
+    tag: {{ .values.node_version }}-vf-2
 
   code-test-executor: &code-test-node-executor
     name: vfcommon/code-test-executor-node-20
     tag: {{ .values.node_version }}-vf-1
 
 orbs:
-  vfcommon: voiceflow/common@{{ .values.common_orb_version }}
+  vfcommon: voiceflow/common@dev:068a24ee36016553dd64e18964d9eec61c5e54d8
```
Comment (xavidop) [inline .circleci/continue_config.yml.tmpl:53]: remember to add here the latest version

**Review by xavidop — APPROVED** [review]

File: .circleci/values.test.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "e2e_env_name": "e2e-general-runtime-12c262a",
-  "common_orb_version": "0.56.2",
+  "common_orb_version": "0.67.0",
+  "node_version": "12.18.3",
```
Comment (theProf) [inline .circleci/values.test.json:4]: where did this value come from? it's an odd node version

Comment (edison-vflow) [reply .circleci/values.test.json:4]: @theProf 


Its just a place holder value to get the orb validation script to run correctly


Thats why its in values.test.json.I guess we can make the dummy variable better by giving it the current node version values

**Pull request closed without merging**